bower.json

```diff
@@ -23,11 +23,12 @@
     "components-font-awesome": "~4.4.0",
     "jquery": "~2",
     "lz-string": "https://github.com/pieroxy/lz-string.git#~1.4.x",
-    "ngDialog": "~0.3.x",
-    "underscore": "~1.8.x"
+    "ngDialog": "~0.5.x",
+    "underscore": "~1.8.x",
+    "angularjs-slider": "~2.10.2",
+    "angular-aria": "~1.5.0"
```

Comment:
haven't used `aria` in any projects before, but it seems to be a perfect fit for this. managing a lot of stuff with the keyboard has already been really convenient.